session-4/lecture-4.ipynb

```diff
@@ -4581,8 +4581,8 @@
     "with tf.Session(graph=g) as sess, g.device('/cpu:0'):\n",
     "    res = softmax.eval(feed_dict={\n",
     "        x: img_4d,\n",
-    "        'vgg/dropout_1/random_uniform:0': [[1.0]],\n",
-    "        'vgg/dropout/random_uniform:0': [[1.0]]})[0]\n",
+    "        'vgg/dropout_1/random_uniform:0': [[1.0] * 4096],\n",
+    "        'vgg/dropout/random_uniform:0': [[1.0] * 4096]})[0]\n",
```

Comment:
If I do not do this then the error happens:
`ValueError: Cannot feed value of shape (1, 1) for Tensor 'vgg/dropout_1/random_uniform:0', which has shape '(?, 4096)'`